scripts/GenSongsList.py

```diff
@@ -18,7 +18,7 @@
 l = lambda p: str(os.path.splitext(os.path.basename(p))[0])
 
 # lambda ext is like lambda l, except it returns the file extension
-ext = lambda p: str(os.path.splitext(os.path.basename(p))[1])
+ext = lambda p: str(os.path.splitext(os.path.basename(p))[1]).lower()
```

Comment:
I modified "ext" to always return a lowercase version of the extension so you wouldn't have to repeatedly call .lower() elsewhere.